packages/components/src/components/indented-text/component.tsx

```diff
@@ -9,6 +9,9 @@ import { Component, h, Host, State } from '@stencil/core';
 	tag: 'kol-indented-text-wc',
 	shadow: false,
 })
+/**
+ * @deprecated This component has been deprecated as it ist just styling.
```

Comment:
Vorschlag:

```suggestion
 * @deprecated Will be removed in v3. Replace with custom HTML and styling.
```
Ich beziehe mich hier auf die bestehende Konvention "Will be removed in v3." und die Beschreibung aus dem Ticket.